src/common/notification/NotificationComponent.tsx

```diff
@@ -18,16 +19,17 @@ function NotificationComponent(props: Props): React.ReactElement | null {
   }
   return (
     <div className={styles.notification}>
-      <DismissableNotification
+      <Notification
+        dismissible
```

Comment:
Probably something I don't know about modern React, but how does this `dismissible` line work? Does the value somehow come from `props` or what's going on here?

Reply:
Ahh, now I remember. It's that same as `dismissible=true`, right?

Reply:
Yeah! And linter actually forbids writing dismissible={true}, so it must be a plain "dismissible" which looks like an undefined variable.